inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/instrumentation/context/InspectitContext.java

```diff
@@ -384,13 +391,11 @@ private void readOverridesFromCurrentTagContext() {
 
     private Stream<Map.Entry<String, Object>> getDataAsStream() {
         return Stream.concat(
-                postEntryPhaseDownPropagatedData.entrySet().stream()
-                        .filter(e -> !dataOverwrites.containsKey(e.getKey())),
-                dataOverwrites.entrySet().stream()
-                        .filter(e -> e.getValue() != null));
+                postEntryPhaseDownPropagatedData.entrySet().stream().filter(e -> !dataOverwrites.containsKey(e.getKey())),
```

Comment:
did you intentionally remove the line breaks here?

Reply:
hmm seems so, maybe I tried to optimize here and then reverted it.. Look easier to read for `Stream.concat`, should I revert?

Reply:
Matter of taste, I'm fine with both